tutorials/python/python-video-generation/code/Readme.md

```diff
@@ -0,0 +1,35 @@
+# Local environment set up
+---
+
+## Install Python 3
+
+### Linux and OSX
+Using version manager [asdf-vm](https://asdf-vm.com/#/core-manage-asdf-vm?id=install-asdf-vm):
+
+Install [Python Plugin](https://github.com/danhper/asdf-python):
+`$ asdf plugin-add python https://github.com/danhper/asdf-python.git`
+
+Install Python:
+`$ asdf install python 3.6.5`
+
+Note: This can be installed using package managers as well, the instruction
+may vary according to your package manager:
+- In Ubuntu: `sudo apt-get install python3`
+- In OSx: `brew install python3`
+
+### Windows
+Using package manager [Chocolatey](https://chocolatey.org/)
+
+`C:\> choco install python3`
+
+#### Install dependencies
+
+From root project directory run `$ pipenv install`
+
+#### Enable your virtual environment
+
+To Enable type `pipenv shell`
+
+#### Disable Virtual Environment
+To Disable it type `exit`
+
```

Comment:
@Larox Great README! Could you please also add a step to execute the script that generates the video? I suppose it would be:
```python
$ pipenv run python python_video.py
```